packages/grid/x-grid-modules/src/components/styled-wrappers/GridOverlay.tsx

```diff
@@ -1,9 +1,10 @@
 import * as React from 'react';
 import { classnames } from '../../utils';
-import { DivProps } from './grid-root';
 import { OptionsContext } from '../options-context';
 
-export function GridOverlay(props: DivProps) {
+type GridOverlayProps = React.HTMLAttributes<HTMLDivElement>;
+
+export function GridOverlay(props: GridOverlayProps) {
```

Comment:
One this component, it seems that we can kill the `<div className="MuiDataGrid-overlayContent" />` element. It will make customization easier.

Reply:
Not sure, it's used in creating the no row message and error-message, and potentially more components.

Reply:
I'm saying that because I didn't see something in the CSS that would prevent. We will see, I could be wrong. It needs to be done in an isolated pull request anyway.